src/targets/gpu/kernels/include/migraphx/kernels/types.hpp

```diff
@@ -25,11 +25,49 @@
 #define MIGRAPHX_GUARD_AMDMIGRAPHX_KERNELS_TYPES_HPP
 
 #include <migraphx/kernels/hip.hpp>
+#include <initializer_list>
```

Comment:
This should not be included when using hiprtc.

Reply:
Removed.